internal/declcfg/testdata/invalid/not-json.txt

```diff
@@ -1 +0,0 @@
-This is not json.not-json.txt
```

Comment:
is this file now empty? did we mean to remove it entirely?

Reply:
I thought I renamed it. I'll double check this.

Reply:
Yeah, this file is removed (actually renamed to `not-yaml-or-json.txt`)